README.md

```diff
@@ -13,19 +13,39 @@ Inline with the latest Wagtail [Long Term Support release](https://github.com/wa
 ## Getting started
 
 If you'd like to contribute to IoGT development, the best way to get started is to clone a copy of this repository and run the code on your machine.
-
-A minimal set of instructions for achieving this are:
-
 ```
-mkdir iogt-dev
-cd iogt-dev
 git clone https://github.com/unicef/iogt.git
-python3 -m venv venv
-source ./venv/bin/activate
-
 cd iogt
+```
+
+Create an [isolated Python environment][1], into which we will install the project's dependencies. This is highly recommended so that the requirements of this project don't interfere with any other Python environments you may have in your system, including the global Python environment. We aso recommend that you create a new environment outside of the project directory. 
+```
+python3 -m venv <path_to_venv_dir>
+source ./venv/bin/activate
```

Comment:
Add custom path to second venv command:
`source <path_to_venv_dir>/bin/activate`